tests/integration/e2e.bats

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bats
+
+
+function setup() {
+  # Create 3 network namespaces
+  sudo ip netns add ns1
+  sudo ip netns add nsnat
+  sudo ip netns add ns2
+
+  # Connect the namespaces using a veth pair ns0 -- nsnat -- ns1
+  sudo ip link add name veth1 type veth peer name vethnat1
+  sudo ip link set netns ns1 dev veth1
+  sudo ip link set netns nsnat dev vethnat1
+
+  sudo ip link add name veth2 type veth peer name vethnat2
+  sudo ip link set netns ns2 dev veth2
+  sudo ip link set netns nsnat dev vethnat2
+
+  # Configure the namespaces network so they can reach each other
+  sudo ip netns exec ns1 ip link set up dev lo
+  sudo ip netns exec ns1 ip link set up dev veth1
+  sudo ip netns exec ns1 ip -6 addr add 2001:db8::1/112 dev veth1
+  sudo ip netns exec ns1 ip -6 route add default via 2001:db8::2
+
+  sudo ip netns exec nsnat ip link set up dev lo
+  sudo ip netns exec nsnat ip link set up dev vethnat1
+  sudo ip netns exec nsnat ip -6 addr add 2001:db8::2/112 dev vethnat1
+
+  sudo ip netns exec nsnat ip link set up dev vethnat2
+  sudo ip netns exec nsnat ip addr add 1.1.1.1/24 dev vethnat2
+
+  sudo ip netns exec ns2 ip link set up dev lo
+  sudo ip netns exec ns2 ip link set up dev veth2
+  sudo ip netns exec ns2 ip addr add 1.1.1.2/24 dev veth2
+
```

Comment:
@siwiutki FYI, got it working, still need to understand what is the problem of executing inside namespace but works out of the framework

Reply:
What is the error message when you execute inside a namespace?

Reply:
just run it locally, it keeps hanging when executed the command, however, if you do the steps manually outside of the framework it works.
I'm suspicios of some bashism thing but didn't have more time to investigate, let me open an issue to track this